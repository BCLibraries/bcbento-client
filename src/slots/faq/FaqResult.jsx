import React from 'react';

function FaqResult({result}) {
    return <li><a href={result.url}>{result.question}</a></li>;
}

export default FaqResult;