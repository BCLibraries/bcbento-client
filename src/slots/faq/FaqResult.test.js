import renderer from 'react-test-renderer';
import FaqResult from './FaqResult.jsx';

it('renders correctly', () => {
    const result = {
        "id": 136455,
        "question": "How do I get free passes to the MFA, the Gardner, and other museums?",
        "url": "https://answers.bc.edu/faq/136455",
    };
    const component = renderer.create(
        <FaqResult result={result}/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
