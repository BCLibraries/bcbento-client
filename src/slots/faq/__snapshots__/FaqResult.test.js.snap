// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li>
  <a
    href="https://answers.bc.edu/faq/136455"
  >
    How do I get free passes to the MFA, the Gardner, and other museums?
  </a>
</li>
`;
